airflow/contrib/hooks/sqoop_hook.py

```diff
@@ -179,11 +179,16 @@ def _import_cmd(self, target_dir, append, file_type, split_by, direct,
         if driver:
             cmd += ["--driver", driver]
 
+        if extra_import_options:
+            for key, value in extra_import_options.items():
+                cmd += ['--{}'.format(key)]
+                if value: cmd += [value]
+
         return cmd
 
-    def import_table(self, table, target_dir, append=False, file_type="text",
+    def import_table(self, table, target_dir=None, append=False, file_type="text",
```

Comment:
`target_dir` should be optional

add the option to pass in extra import arguments that sqoop offers. There are many options that sqoop offers that aren't part of the SqoopOperator and there should be a way to pass those in.

Reply:
👍